notebooks/starter_notebook.py

```
@@ -0,0 +1,260 @@
+
+    Returns:
+    None. Displays the ROC curve plot.
+    """
+    plt.figure(figsize=(10, 8))
+    for model, x_val, y_val in zip(models, vals_x, vals_y):
+        if isinstance(model, CatBoostClassifier):
+            y_score = model.predict_proba(x_val)[:, 1]
+            fpr, tpr, _ = roc_curve(y_val, y_score)
+            roc_auc = auc(fpr, tpr)
+            plt.plot(fpr, tpr, lw=2, label=f'{model.__class__.__name__} (area = {roc_auc:.2f})')
+    
```

Ruff

Blank line contains whitespace

(W293)